en/docs/api-analytics/on-prem/datadog.md

```diff
@@ -0,0 +1,157 @@
+       "support_rules": "",
+       "match_rules": "rules_1 %{date(\"HH:mm:ss,SSS\")}%{regex(\"[^a-zA-Z0-9]*\\\\w+[^a-zA-Z0-9]*\\\\w+[^a-zA-Z0-9]*\\\\w+ \\\\w+ [a-zA-Z0-9]*[^a-zA-Z0-9]*\\\\w+[^a-zA-Z0-9]\")}%{regex(\"\\\\w+\"):eventType}%{regex(\"[^a-zA-Z0-9]* \\\\w+ [^a-zA-Z0-9]\")}%{data::json}\n"
+     },
+     "name": "Grok Parser",
+     "source": "message",
+     "samples": ["08:23:34,898 [-] [PassThroughMessageProcessor-188]  INFO ELKCounterMetric apimMetrics: apim:response, properties :{\"apiName\":\"PizzaShackAPI\",\"proxyResponseCode\":200,\"destination\":\"https://localhost:9443/am/sample/pizzashack/v1/api/\",\"apiCreatorTenantDomain\":\"carbon.super\",\"platform\":\"Other\",\"apiMethod\":\"GET\",\"apiVersion\":\"1.0.0\",\"gatewayType\":\"SYNAPSE\",\"apiCreator\":\"admin\",\"responseCacheHit\":false,\"backendLatency\":3,\"correlationId\":\"02951f10-5934-456d-b608-7cd85b3f7c9d\",\"requestMediationLatency\":1,\"keyType\":\"SANDBOX\",\"apiId\":\"af88b5f8-594a-4f57-a729-7eb86b6a63ab\",\"applicationName\":\"DefaultApplication\",\"targetResponseCode\":200,\"requestTimestamp\":\"2022-12-15T02:53:34.894Z\",\"applicationOwner\":\"admin\",\"userAgent\":\"curl\",\"userName\":\"admin@carbon.super\",\"apiResourceTemplate\":\"/menu\",\"responseLatency\":4,\"regionId\":\"default\",\"responseMediationLatency\":0,\"userIp\":\"127.0.0.1\",\"apiContext\":\"/pizzashack/1.0.0\",\"applicationId\":\"87c8da4c-aa3f-43d3-89ac-dc3dc3b19d3e\",\"apiType\":\"HTTP\",\"properties\":{}}"],
+     "type": "grok-parser"
+   }
+ ]
+}'
+
+```
+
+After creating the above pipeline, invoke a few APIs through API Manager to generate few Analytics events(both Success and Faulty events have to be generated). 
+
+The published analytics events will be available in `Logs -> Search` view on the DataDog web UI.
+
+#### Step 2.3 - Create Facets and Mesures.
+
+1. For the following attributes, create facets by following this [documentation.](https://docs.datadoghq.com/logs/explorer/facets/#create-facets)
+
+    ```
+    apiContext
+    apiCreator
+    apiName
+    apiType
+    apiVersion
+    apiResourceTemplate
+    apiMethod
+    applicationName
+    applicationOwner
+    errorType
+    eventType
+    Destination
+    Platform
+    responseCacheHit
+    userAgent
+    userName
+    userIp
+    ```
+
+2. Create measure for following attributes
+
+    ```
+    backendLatency - Type Integer - Unit Milliseconds
+    responseLatency - Type Integer - Unit Milliseconds
+    responseMediationLatency - Type Integer - Unit Milliseconds
+    requestMediationLatency - Type Integer - Unit Milliseconds
+
+    targetResponseCode - Type Integer - Unit None
+    proxyResponseCode - Type Integer - Unit None
+    errorCode - Type Integer - Unit None
+    ```
+
+#### Step 2.4 - Import Dashboards
+
+Import the [dashboard](https://drive.google.com/drive/folders/1hEm__ODR3zA5slCCKY2TOcFnKmB0OTnV) following the steps [here.](https://docs.datadoghq.com/dashboards/#copy-import-or-export-dashboard-json)
```

Comment:
Is it necessary to add the Gdrive link, because we don't add such as a practice. If you need to add an attachment, you can do it in the doc repo itself. Please ping me if you need to know how.

Reply:
Done